spec/schemas/ml._common.yaml

```diff
@@ -51,20 +89,75 @@ components:
         - relation
         - value
     SearchModelsHitsHit:
+      allOf:
+        - $ref: '#/components/schemas/BaseResponse'
+        - type: object
+          properties:
+            _index:
+              type: string
+              description: The index.
+            _id:
+              type: string
+              description: The hit ID.
+            _score:
+              type: number
+              description: The score.
+            _source:
+              $ref: '#/components/schemas/Source'
+            model_id:
+              type: string
+              description: The model ID.
+          required:
+            - _score
+    Source:
       type: object
       properties:
-        _index:
+        last_deployed_time:
+          type: number
+          description: The last deployed time.
+        model_version:
           type: string
-          description: The index.
-        _id:
+          description: The model version.
+        created_time:
+          type: number
+          description: The created time.
+        deploy_to_all_nodes:
+          type: boolean
+          description: Whether to deploy to all nodes.
+        is_hidden:
+          type: boolean
+          description: Whether the model is hidden.
+        description:
           type: string
-          description: The hit ID.
-        model_id:
+          description: The model description.
+        model_state:
```

Comment:
Is this an enum?

Reply:
Right, this is an enum

Reply:
Enumerate the values if it's a finite list.